_hackbars/belamo.md

```diff
@@ -0,0 +1,15 @@
+---
+Layout: hackbar
+Title:  Climate change is real
+Author: Raymond Belamo
+---
+
+
+When you get into developing, you would'nt really understand open source until you start writing codes. Then You will discover the power in the open source community. 
+
+
+---
+
+## People need to stop the climate change denial. It's as real as it gets. It's high time we understand that renewables are the key to securing a better planet. Climate change is real!
```

Comment:
Codacy found an issue: [[no-heading-punctuation] Don’t add a trailing `!` to headings](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6302588)